asgiref/sync.py

```diff
@@ -15,6 +15,29 @@
     contextvars = None
 
 
+def _restore_context(context):
```

Comment:
Since these two functions are only used by AsyncToSync, could you move them underneath it as classmethods or similar?

Reply:
`_restore_context` is shared across the two classes, what's your preference there?